src/constraints.jl

```diff
@@ -152,14 +152,35 @@ function MOI.get(m::LinQuadOptimizer, ::Type{MOI.ConstraintIndex{F,S}}, name::St
     m.constraint_names_rev[name]::MOI.ConstraintIndex{F,S}
 end
 
+
+"""
+    CSRMatrix{T}
+
+Matrix given in compressed sparse row (CSR) format.
+
+It contains three vectors:
+ - `rowptr` is a vector pointing to the start of each row in
+    `colval` and `nzval`;
+ - `colval` is a vector of column numbers; and
+ - `nzval` is a vector of corresponding nonzero values.
+
+The length of `rowptr` is the number of rows in the matrix.
+
+`CSRMatrix` is analgous to the structure in Julia's `SparseMatrixCSC` but with
```

Comment:
Eh, the rename doesn't help with readability. `rowptr` is worse than `row_pointers`. I'd revert this and just mention the mapping to `SparseMatrixCSC`.